src/IAM/Infrastructure/Persistence/MySQL/Repository/MySQLRefreshToken.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Twitter\IAM\Infrastructure\Persistence\MySQL\Repository;
+
+use DateTimeImmutable;
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\ParameterType;
+use Twitter\IAM\Application\Logout\RefreshTokenRepository;
+use Twitter\IAM\Domain\Auth\Exception\TokenInvalidException;
+use Twitter\IAM\Infrastructure\Auth\RefreshTokenHasher;
+use Twitter\Shared\Infrastructure\Persistence\Doctrine\UuidBinaryConverter;
+
+final class MySQLRefreshToken implements RefreshTokenRepository
```

Comment:
```suggestion
final class MySQLRefreshTokenRepository implements RefreshTokenRepository
```

@SergeyGrinenko, be consistent with naming.